File: CallBackDemo/app/src/main/java/com/example/tuozhaobing/callbackdemo/MainActivity.java
package com.example.tuozhaobing.callbackdemo;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity implements OnLoginInforCompleted{

    private TextView mContentTextView;
    private CustomDialogFragment mCustomDialogFragment;
    public static final String COMMAND = "USER_INFOR_DIALOG";
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        mContentTextView = (TextView) findViewById(R.id.user_input_content);
        mCustomDialogFragment = new CustomDialogFragment();
        mCustomDialogFragment.setOnLoginInforCompleted(this);
        mContentTextView.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                mCustomDialogFragment.show(getFragmentManager(),COMMAND);
            }
        });
    }

    @Override
    public void inputLoginInforCompleted(String userName, String passWord) {
        mContentTextView.setText(userName+"::::"+passWord);
    }
}

File: TouchEventTest/app/src/main/java/com/example/administrator/toucheventtest/MainActivity.kt
package com.example.administrator.toucheventtest

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.view.MotionEvent

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    override fun dispatchTouchEvent(ev: MotionEvent?): Boolean {
        return super.dispatchTouchEvent(ev)
    }
}
